db/fixtures/miq_product_features.yml

```diff
@@ -823,6 +823,10 @@
       :description: Remove Volumes
       :feature_type: admin
       :identifier: cloud_volume_delete
```

Comment:
I'm assuming you're reusing this for safe delete also right?

Reply:
Yeah, It should be the same permission because you end up with a deleted volume either way.

Reply:
> @agrare I had to add a new product feature to make the api tests pass